easybuild/easyconfigs/p/ParallelIO/ParallelIO-2.2.2a-intel-2017a.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'ParallelIO'
+version = '2.2.2a'
+
+homepage = 'https://github.com/NCAR/ParallelIO'
+description = """A high-level Parallel I/O Library for structured grid applications"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+## underscore = "${version//./_}"           something like this  ??
+## sources = ['pio%(underscore)s.tar.gz']   bash sub in easyconf ??
+sources = ['pio2_2_2a.tar.gz']
+source_urls = ['https://github.com/NCAR/ParallelIO/archive/']
+
+separate_build_dir = True
+
+dependencies = [
+  ('PnetCDF', '1.8.1', '', ('intel', '2017a')),
+  ('netCDF-Fortran', '4.4.4', '', ('intel', '2017a'))
+]
+
+builddependencies = [
+  ('CMake', '3.8.2', '', ('intel', '2017a')),
```

Comment:
This version of `CMake` does not exist in the `easyconfigs` repository, so you either need to provide it yourself, or change it to one of those that does exist e.g. `CMake-3.8.2-GCCcore-6.3.0.eb`.